apps/autoscheduler/autotune_loop.sh

```diff
@@ -204,12 +203,13 @@ for ((BATCH_ID=$((FIRST+1));BATCH_ID<$((FIRST+1+NUM_BATCHES));BATCH_ID++)); do
         # benchmark them serially using rungen
         for ((SAMPLE_ID=0;SAMPLE_ID<${BATCH_SIZE};SAMPLE_ID++)); do
             S=$(printf "%d%02d" $BATCH_ID $SAMPLE_ID)
-            benchmark_sample "${DIR}/${SAMPLE_ID}" $S $EXTRA_ARGS_IDX
+            FNAME=$(printf "%s_batch_%02d_sample_%02d" ${PIPELINE} $BATCH_ID $SAMPLE_ID)
```

Comment:
There are often going to be many more than 100 batches. I'd use %04d for the batch id

Reply:
OK, but this line is an exact copy of line 198, so that limitation has already been in place for months. I'll make the change both places.